functions.test.js

```diff
@@ -0,0 +1,20 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictEquals = require('.\\functions')
+
+
+describe('Functions', () => {
+    describe('strictEquals', () => {
+        it('checks for equality and returns a boolean', () => {
```

Comment:
We would expect four test scenarios here

1. Two parameters with the same data type and same value (ex. 5 and 5)
2. Two parameters with the same data type but different value (ex. 5 and 6)
3. Two parameters with a different data type but same value (ex. 5 and '5')
4. Two parameters with a different data type and different value (ex. 5 and 'a')